stack.yaml

```diff
@@ -1,10 +1,11 @@
-resolver: lts-13.14
+resolver:
+  url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/17/5.yaml
+
 packages:
-- .
+  - .
+
 ghc-options:
-  "$locals":
-    -fhide-source-paths
-    -Weverything
+  "$locals": -fhide-source-paths
```

Comment:
Was removing `-Weverything` intentional?

Reply:
Yep, I was getting a bunch of very wacky failures that I was having trouble sorting through.